src/system/ExtensionsModule/Resources/docs/ExtensionsInterfaceController_and_Twig_tags.md

```diff
@@ -0,0 +1,53 @@
+ExtensionsInterfaceController
```

Comment:
I think it is a good place to start think about that helpAction and how it can be used to help arrange and display helps/docs/manuals for module without dropping everything to one folder it can for example scan docs/ folder for each module, display file names and links to load docs(md) all via ajax for user, admin, dev. I can do working example :)